config/templates/stack-template.json

```diff
@@ -742,7 +754,13 @@
         "SecurityGroups" : [
           { "Ref" : "SecurityGroupElbAPIServer" }
         ]
-      }
+      },
+      "DependsOn" : [
+        {{range $etcdIndex, $etcdInstance := .EtcdInstances}}
+        {{if gt $etcdIndex 0}},{{end}}
+        "InstanceEtcd{{$etcdIndex}}"
+        {{end}}
+      ]
```

Comment:
That explains it.  Was using the latest release at the time of v0.9.3-rc.5.  Will update and re-test.